src/components/views/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,137 @@
+import { Box, Drawer, Popper, Portal, Typography } from '@mui/material';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import InstancesTreeView from './TreeView/InstanceTreeView';
+
+const datasets = [
+  {
+    id: uuidv4(),
+    label: 'Stimulus',
+    tooltip: 'Stimulus',
+    type: 'MECHANISM',
+  },
+  {
+    id: uuidv4(),
+    label: 'Projection Stimulus --> Composition',
+    tooltip: 'Projection Stimulus --> Composition',
+    type: 'PROJECTION',
+  },
+  {
+    id: uuidv4(),
+    label: 'Composition 1 Composition 1Composition 1',
+    tooltip: 'Composition 1',
+    type: 'COMPOSITION',
+    items: [
+      {
+        id: uuidv4(),
+        label: 'A1',
+        tooltip: 'Stimulus',
+        type: 'MECHANISM',
+      },
+      {
+        id: uuidv4(),
+        label: 'Projection A1 -->  B1',
+        tooltip: 'Projection A1 --> B1',
+        type: 'PROJECTION',
+      },
+      {
+        id: uuidv4(),
+        label: 'B1',
+        tooltip: 'Stimulus',
+        type: 'MECHANISM',
+      },
+    ],
+  },
+  {
+    id: uuidv4(),
+    label: 'Composition 2',
+    tooltip: 'Composition 2',
+    type: 'COMPOSITION',
+    items: [
+      {
+        id: uuidv4(),
+        label: 'A2',
+        tooltip: 'Stimulus',
+        type: 'MECHANISM',
+      },
+      {
+        id: uuidv4(),
+        label: 'Projection A2 -->  B2 Projection A2 -->  B2',
+        tooltip: 'Projection A2 --> B2',
+        type: 'PROJECTION',
+      },
+      {
+        id: uuidv4(),
+        label: 'B2',
+        tooltip: 'Stimulus',
+        type: 'MECHANISM',
+        items: [
+          {
+            id: uuidv4(),
+            label: 'A1',
+            tooltip: 'Stimulus',
+            type: 'MECHANISM',
+          },
+          {
+            id: uuidv4(),
+            label: 'Projection A1 -->  B1',
+            tooltip: 'Projection A1 --> B1',
+            type: 'PROJECTION',
+          },
+          {
+            id: uuidv4(),
+            label: 'B1',
+            tooltip: 'Stimulus',
+            type: 'MECHANISM',
+          },
+        ],
+      },
+    ],
+  },
+];
+
+const sidebarOpened = true;
+
+export const Sidebar = (props) => {
+  const { expand, setExpand, searchTerm } = props;
+
+  const renderContent = () => {
+    if (datasets?.length > 0) {
+      return (
+        <>
+          <Box paddingLeft="1.5rem" paddingY={1.5}>
```

Comment:
@emekauja Incorrect padding should be an px passing instead of just left. Left would 2 not 1.5, so px={2}